src/addTask.js

```diff
@@ -0,0 +1,11 @@
+// import displayTodo from './displayTasks.js';
+import getTasks from './getTasks.js';
+
+const addTask = (task) => {
+  const todos = getTasks();
+  todos.push(task);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  // displayTodo();
+};
```

Comment:
- [ ] The property index should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). please address that!
![task](https://user-images.githubusercontent.com/44745855/185660228-20be9a23-0153-405b-a242-c2f14205c9cf.png)

Reply:
- [ ] Make sure after deleting a task you should update all remaining items' indexes, so they represent the current list order and are unique.